Add files via upload

UshnahAbbasi opened the pull request:
1. I have used .env file to set environment variables (username and password) for making them secure instead of writing them in .py file.
2. modular approach has been used (another .py file has the function to make connection to the database)
3. all information about API (in/out) and database has been added to the readme file

UshnahAbbasi committed:
Add files via upload

umerkhalidarbisoft commented:
Can you please do the following things
1. improve code using modular approach? e.g. db access separation from the 
2. How can you make application little secure so that you don't have to write username/passwords in the code? 
3. Provide an info how to query the API? for example what is in/out

UshnahAbbasi committed:
Update API_mem_usage.py

UshnahAbbasi committed:
Create db_connect.py

UshnahAbbasi committed:
Update db_connect.py

made some supporting functions for all the tasks associated with database.

UshnahAbbasi committed:
Update API_mem_usage.py

Added 3 more columns named hour, minute, second. 
if the timestamp is present in the parameters of request, the exact data row is the output, else if hour or minute or second is given the rows are given as per required. If that particular data doesn't exist in the database, a message saying it doesn't exist is displayed as an output of the request.

UshnahAbbasi commented:
1. I have added the commits from the other request here in this one too and will be deleting that one.
2. three separate columns have been made
3. the functions of database have been separated
4. updated the readme file according to the new request format and respective outputs.

UshnahAbbasi committed:
Update API_mem_usage.py

UshnahAbbasi committed:
Update db_connect.py

UshnahAbbasi commented:
I have updated the files doing the following:
1. the response of API has been altered such that it has keys now too.
2. the API has been converted to be of POST method.
3. the readme file has been updated

umerkhalidarbisoft commented on db_connect.py:
```diff
@@ -0,0 +1,47 @@
+        if minute != None:
+            query = f"SELECT * FROM memory_usage WHERE hour = {hour} and minute ={minute}"
+        if second != None:
+            query = f"SELECT * FROM memory_usage WHERE hour = {hour} and minute ={minute} and second={second} "
```
what if I have provided only the "second" value and not the "hour" & "minute" values?

UshnahAbbasi committed:
Update db_connect.py

UshnahAbbasi committed:
Update db_connect.py
